Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -309,6 +309,29 @@ QString GameFile::GetLanguage(DiscIO::Language lang) const
   }
 }
 
+QString GameFile::GetFileType() const
+{
+  switch (GetBlobType())
+  {
+  case DiscIO::BlobType::PLAIN:
+    return QObject::tr("Plain");
```

Comment:
In addition to what @mbc07 said, I think Plain can be tricky to translate well. It would be nice if we could switch between ISO/WAD/whatever based on the volume type.